bentoml/_internal/configuration/__init__.py

```diff
@@ -2,14 +2,23 @@
 import os
 import typing as t
 
+from ...exceptions import BentoMLException
+
 try:
     import importlib.metadata as importlib_metadata
 except ModuleNotFoundError:
     import importlib_metadata
 
 import yaml
 
-import bentoml._version as version_mod
+try:
+    import bentoml._version as version_mod
+except ImportError:
+    # Since we aren't VCS _version.py, which is generated by setuptools_scm
```

Comment:
when will this happen? 

I suppose only when a developer does `pip install -e .` and then removed the `_version.py` file by accident?

Reply:
yes. Also mainly for type checking where pyright cannot find `bentoml._version`.
There is also a current caveats with types CI atm